products/cloudrun/terraform.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  DomainMapping: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloudrun_domain_mapping_basic"
+        min_version: 'beta'
+        primary_resource_id: "default"
+        vars:
+          cloudrun_domain_name: "tftest-domainmapping.com"
+        test_env_vars:
+          namespace: :PROJECT_NAME
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      encoder: 'templates/terraform/encoders/cloudrun.go.erb'
+      decoder: 'templates/terraform/decoders/cloudrun.go.erb'
+    properties:
+      name: !ruby/object:Overrides::Terraform::PropertyOverride
+        required: true
+        exclude: false
+      # excluding kind from Terraform code because it only accepts the same single value
+      # so we hard code it in the encoder instead.
+      kind: !ruby/object:Overrides::Terraform::PropertyOverride
+        exclude: true
+      apiVersion: !ruby/object:Overrides::Terraform::PropertyOverride
+        output: true
```

Comment:
tbh - I'm worried that this value might be different even when querying the same endpoint. I am pretty sure that you will be able to retrieve Services created at v1alpha1 via the v1beta1 endpoint when it ships. So I thought it would be valuable down the road.

Reply:
Since it's easier to add than to remove, would it make sense to wait until it becomes valuable to add it?

Reply:
Since it's easier to add than to remove, would it make sense to wait until it becomes valuable to add it?

Reply:
sound good.